spec/features/channel_filter_on_results_page_spec.rb

```diff
@@ -0,0 +1,175 @@
+      and_i_submit_the_remote_search_form
+      then_i_am_on_results_page
+      and_i_see_remote_advice_method_selected
+      and_only_remote_firms_are_displayed
+    end
+  end
+
+  scenario 'Consumer changes advice method from in person to remote' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_have_performed_an_in_person_search_from_the_landing_page
+      when_i_select_remote_advice
+      and_i_submit_the_sidebar_search_form
+      then_i_am_on_results_page
+      and_i_see_remote_advice_method_selected
+      and_only_remote_firms_are_displayed
+    end
+  end
+
+  scenario 'Consumer changes advice method from remote to in person' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_have_performed_a_remote_search_from_the_landing_page
+      when_i_select_face_to_face_advice
+      and_i_submit_the_sidebar_search_form
+      then_i_am_on_results_page
+      and_i_see_face_to_face_advice_method_selected
+      and_only_in_person_firms_are_displayed
+    end
+  end
+
+  scenario 'Consumer changes advice method to in person without providing postcode' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_have_performed_a_remote_search_from_the_landing_page
+      when_i_select_face_to_face_advice_without_providing_postcode
+      and_i_submit_the_sidebar_search_form
+      then_i_am_on_the_landing_page
+      and_a_validation_error_message_is_displayed
+    end
+  end
+
+  scenario 'Consumer changes advice method to remote without selecting remote advice type' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_have_performed_an_in_person_search_from_the_landing_page
+      when_i_select_remote_advice_without_providing_type
+      and_i_submit_the_sidebar_search_form
+      then_i_am_on_the_landing_page
+      and_a_validation_error_message_is_displayed
+    end
+  end
+
+
+  def given_some_firms_were_indexed
+    with_fresh_index! do
+      @phone_firm = create(:firm, registered_name: 'The Willers Ltd', in_person_advice_methods: [], other_advice_methods: [ phone_advice ])
+      create(:adviser, postcode: 'RG2 8EE', firm: @phone_firm,       latitude: 51.428473, longitude: -0.943616)
+
+      @online_firm = create(:firm, registered_name: 'The Equiters Ltd', in_person_advice_methods: [], other_advice_methods: [ online_advice ])
+      create(:adviser, postcode: 'LE1 6SL', firm: @online_firm, latitude: 52.633013, longitude: -1.131257)
+
+      @in_person_firm = create(:firm, registered_name: 'Estate Traders Ltd', other_advice_methods: [])
+      create(:adviser, postcode: 'G1 5QT', firm: @in_person_firm, latitude: 55.856191, longitude: -4.247082)
+    end
+  end
+
+  def and_i_am_on_the_rad_landing_page
+    landing_page.load
+  end
+
+  def and_i_have_performed_an_in_person_search_from_the_landing_page
+    landing_page.load
+    landing_page.in_person.tap do |section|
+      section.postcode.set('G1 5QT')
```

Comment:
It's a total nit-pick but we don't use parens for any of the site prism setters throughout the other feature specs. Can we make this consistent with elsewhere. I know, I know. I am the consistency police :police_car: :cop: